### PR TITLE
Remove documentation for wptrunner as a standalone project

### PR DESCRIPTION
**Pull request by foolip**
wptrunner 1.14 was released in 2015, and while it can be installed in a
virtualenv, these instructions should not be followed by anyone these
days.

Integrate some of it into from-local-system.md instead.

### CODE REVIEW

**Review by jgraham — APPROVED** [review]
The installing part is definitely wrong/outdated, but it's possible that the usage parts aren't really documented elsewhere. Could be worth saving some of that if you have time.

**Comment by foolip** [comment]
@jgraham I'll comment on the bits that I thought you'd tell me to move somewhere else :)

File: tools/wptrunner/docs/usage.rst [hunk 1]
```diff
@@ -1,242 +0,0 @@
...
-
-The ``MANIFEST.json`` file needs to be regenerated each time the
-web-platform-tests checkout is updated. To aid with the update process
-there is a tool called ``wptupdate``, which is described in
-:ref:`wptupdate-label`.
-
-Running the Tests
------------------
-
-A test run is started using the ``wptrunner`` command.  The command
```
Comment (foolip) [inline tools/wptrunner/docs/usage.rst:53]: The current documentation for this is the output of `./wpt run --help`, extracted into https://web-platform-tests.org/running-tests/command-line-arguments.html#run.

Some arguments are documented a bit more explicitly here.

File: tools/wptrunner/docs/usage.rst [hunk 2]
```diff
@@ -1,242 +0,0 @@
...
-subdirectory, specify the directory name in the positional arguments after
-the options; for example, run just the tests in the `dom` subdirectory::
-
-  wptrunner --metadata=~/testtwf/meta --tests=~/testtwf/web-platform-tests/ \
-    --binary=/path/to/firefox --certutil-binary=/path/to/certutil \
-    --prefs-root=/path/to/testing/profiles \
-    dom
-
--------------------
-Running in parallel
```
Comment (foolip) [inline tools/wptrunner/docs/usage.rst:137]: This is a better description of `--processes` than exists anywhere else, I think. I'll integrate it into `from-local-system-md`.

File: tools/wptrunner/docs/usage.rst [hunk 3]
```diff
@@ -1,242 +0,0 @@
...
-See the `Configuration File`_ section for more information about
-configuration files, including information about their expected
-locations.
-
-The content of the ``wptrunner.default.ini`` default configuration file
-makes wptrunner look for tests (that is, a web-platform-tests checkout)
-as a subdirectory of the current directory named ``tests``, and for
-metadata files in a subdirectory of the current directory named ``meta``.
-
-Output
```
Comment (foolip) [inline tools/wptrunner/docs/usage.rst:164]: Some of this is worth retaining as well, but needs to be rewritten quite a bit.

**Comment by foolip** [comment]
@jgraham do you want to review the salvaged (half rewritten) documentation?

**Review by sideshowbarker — APPROVED** [review]

File: docs/running-tests/from-local-system.md [hunk 4]
```diff
@@ -152,7 +152,7 @@ customising the test run:
     ./wpt run --help
 
 [A complete listing of the command-line arguments is available
-here](command-line-arguments.md).
+here](command-line-arguments.md#run).
```
Comment (foolip) [inline docs/running-tests/from-local-system.md:155]: I checked the build output and this actually broke stuff:
```diff
-<p><a class="reference internal" href="command-line-arguments.html"><span class="doc">A complete listing of the command-line arguments is available
-here</span></a>.</p>
+<p><a class="reference external" href="command-line-arguments.md#run">A complete listing of the command-line arguments is available
+here</a>.</p>
```

Comment (foolip) [reply docs/running-tests/from-local-system.md:155]: I found there are a few cases of broken links like this already that also need fixing...

Comment (foolip) [reply docs/running-tests/from-local-system.md:155]: Sent https://github.com/web-platform-tests/wpt/pull/28831, will apply the same fix here.

**Review by jgraham — APPROVED** [review]

**Pull request merged**